.gitignore

```diff
@@ -1,15 +1,26 @@
+
 # Binaries for programs and plugins
 *.exe
 *.exe~
 *.dll
 *.so
 *.dylib
+bin
 
-# Test binary, built with `go test -c`
+# Test binary, build with `go test -c`
```

Comment:
This excludes binaries that are built with `go test -c`. It does not build them.